website/docs/compute/local-deployment/local-deployment.md

```diff
@@ -0,0 +1,219 @@
+# Local Deployment
+
+It’s possible to deploy a Compute environment to a local environment using a local Kubernetes cluster.
+
+## Requirements
+
+- [Docker](https://www.docker.com/)
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- [ngrok](https://ngrok.com/download)
+- [subo](https://github.com/suborbital/subo#installing)
+
+## Steps
+
+### 1. Create a folder for the environment
+
+This is a temporary place where we’ll create and configure our Compute environment:
+
+```bash
+mkdir my-compute; cd my-compute
+```
+
+### 2. Start up our Kubernetes cluster
+
+Kubernetes clusters usually live on the cloud. However, with minikube, we can create a local one to use:
+
+```bash
+minikube start
+```
+
+### 3. Expose our cluster to the internet with ngrok
+
+This command will forward all requests to a randomly-generated URL to `http://localhost:80`
+
+```bash
+ngrok http http://localhost
+```
+
+> **Note:** Jot down that URL generated by ngrok! It’ll look something like [https://84925795ffae.eu.ngrok.io](https://84925795ffae.eu.ngrok.io/)
+
+### 4. Generate your Compute manifests
+
+Next we’ll be using subo to generate our Kubernetes manifest files!
+
+```bash
+subo compute deploy core --dryrun
+```
+
+You will be asked for a domain. Please make sure to enter your domain from ngrok.
+
+These will now live in the `.suborbital/` folder.
+
+### 5. Disable TLS checks in the Compute environment
+
+Open up `.suborbital/scc-controlplane-deployment.yaml` in your editor of choice, and make the following changes:
+
+Under the Builder Container (Line 51):
+
+```yaml
+- name: builder
+          image: suborbital/scc-builder:v0.3.1
+          command: ["builder"]
+
+          ports:
+            - containerPort: 8080
+            - containerPort: 8443
+
+          env:
+            - name: SCC_DOMAIN
+              value: "<YOUR_NGROK_DOMAIN>"
+
+            - name: SCC_TLS_PORT
+              value: "8443"
+
+            - name: SCC_LOG_LEVEL
+              value: "info"
+
+            - name: SCC_CONTROL_PLANE
+              value: "scc-controlplane-service:8081"
+
+          volumeMounts:
+            - mountPath: "/home/scn"
+              name: controlplane-storage
+```
+
+Delete the following line:
+
+`delete containerPort: 8443`
+
+Delete the following key-value pair:
+
+```yaml
+- name: SCC_DOMAIN
+  value: "<YOUR_NGROK_DOMAIN>"
+```
+
+Replacing the following key-value pair:
+
+```yaml
+- name: SCC_TLS_PORT
+  value: "8443"
+```
+
+With the following:
+
+```yaml
+name: SCC_HTTP_PORT
+value: "8080"
+```
+
+Under the `scc-builder-service` in line 124:
```

Comment:
This line numbers will not age well :grimacing:

Reply:
I think we'll be able to avoid the need for the line numbers—and all of this copy-pasta, actually—if we do https://github.com/suborbital/subo/issues/338. Totally on my list for my 20% dev time! And the line numbers will probably be stable for the next few weeks, I think? So this seems good for now.

Reply:
You're both absolutely right. In fact, as we delete lines in these instructions, the line numbers get jumbled anyway. Thanks!